tensorflow/python/keras/preprocessing/image_test.py

```diff
@@ -345,6 +345,27 @@ def test_img_utils(self):
     x = keras.preprocessing.image.img_to_array(img, data_format='channels_last')
     self.assertEqual(x.shape, (height, width, 1))
 
+    # Test invalid usage
+    # not 3D
+    with self.assertRaises(ValueError):
```

Comment:
When testing `ValueError`, please check the error message against a regex pattern (see other examples in the codebase). This makes sure that you are catching the error you intend to be raised.

Reply:
review comments accepted and updated the changes, please review and thanks.